docs/admin.md

```diff
@@ -0,0 +1,9 @@
+# Admin settings
+
+You can find 'admin' menu when on start page of MOLGENIS, when you view the list of databases (click logo to get there).
+Only when signed in as 'admin' user this menu item will be shown.
+
+## User management
+
+Currently you can use the admin menu to view the users currently registered in the system. In addition you can create a
+new user and/or set a user password.
```

Comment:
nice 👍 , hope we can keep building on the docs , makes it easer to onboard new projects/devs/users